examples/application.properties

```diff
@@ -24,9 +24,16 @@ config.restartability.allow=false
 
 
 # MONGO DATABASE
+spring.data.mongodb.host=
+spring.data.mongodb.authentication-database=
+spring.data.mongodb.username=
+spring.data.mongodb.password=
+spring.data.mongodb.database=
 config.db.read-preference=primary
 db.collections.features.name=features
+db.collections.files.name=files
```

Comment:
It's misleading to put `files.name` and `variants.name` here, as they are not used, they are overwritten with the opencga ones